src/theme/style-overrides/components.ts

```diff
@@ -0,0 +1,16 @@
+import { ThemeOptions } from "@mui/material";
+
+const COMPONENTS_STYLE_OVERRIDES: ThemeOptions["components"] = {
+  MuiButton: {
+    styleOverrides: {
+      root: {
+        paddingInline: "16px",
```

Comment:
Idea (not important now): Perhaps [spacing controls](https://mui.com/material-ui/customization/spacing/) could work?